tests/unit/ArrTest.php

```diff
@@ -169,4 +169,271 @@ public function testExtract($arr)
             );
         $this->assertEquals(Arr::extract($arr['top'], $paths, $default), $expect);
     }
+
+    public function dataMappedContact()
+    {
+        return array(
```

Comment:
We [require PHP 5.4](https://github.com/kohana/ohanzee-helpers/blob/master/composer.json#L30) so please use short array syntax.